src/web-ui/src/components/DemoGuideBadge/DemoGuideBadge.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <button @click="onClick" :aria-label="copy" class="demo-guide-badge d-inline-flex align-items-center text-left">
+    <div class="logo mr-1"><img :src="serviceLogo" alt="" class="img-fluid" /></div>
+    <div :class="{ text: true, 'hide-text-on-small-screens': hideTextOnSmallScreens }">
+      <div>{{ copy }}</div>
+      <div class="powered-by">
+        powered by <span class="service">{{ poweredByService }}</span>
+      </div>
+    </div>
+  </button>
+</template>
+
+<script>
+import { mapActions } from 'vuex';
+
+import { Articles } from '@/partials/AppModal/DemoGuide/config';
+
+const Services = {
+  Pinpoint: 'Pinpoint',
+  Personalize: 'Personalize',
+};
+
+export default {
+  name: 'DemoGuideBadge',
+  props: {
+    article: { type: String, required: true },
+    hideTextOnSmallScreens: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  computed: {
+    service() {
+      switch (this.article) {
+        case Articles.TEXT_MESSAGING:
```

Comment:
NIT: Can we call this SMS_MESSGAING instead of text to keep the naming as close as possible to the feature

Reply:
Doen